source/reference/limitations.txt

```diff
@@ -104,6 +107,18 @@ Sharded Clusters
 - ``mongosync`` only syncs indexes that exist on all shards.
 - ``mongosync`` only syncs indexes that have consistent index
   specifications on all shards.
+- You must stop the balancer on a sharded source cluster until
+  mongosync finishes the collection copy phase.
+
+  .. note::
+
+     After stopping the balancer, wait fifteen minutes before
+     starting ``mongosync``. This gives the cluster time to
+     finish any in progress chunk migrations.
+
+- You can not run the  :dbcommand:`moveChunk` and
+  :dbcommand:`moveRange` commands on the source cluster.
+
 
 .. note::
```

Comment:
The note here references the point ` ``mongosync`` only syncs indexes that have consistent index
  specifications on all shards.`, but it has been pushed down by the added points. The points should be added after this note.

Reply:
Also would be nice to make this note indented with the bullet point.

Reply:
Good catch.  Fixed